lib/service/connect.go

```diff
@@ -792,11 +792,6 @@ func (process *TeleportProcess) newClient(authServers []utils.NetAddr, identity
 	log.Debugf("Attempting to connect to Auth Server directly.")
 	_, err = directClient.GetLocalClusterName()
 	if err != nil {
-		// Only attempt to connect through the proxy for nodes.
```

Comment:
Just a heads up I'll be doing some refactoring around here as well. Application servers will only attempt to dial though the tunnel and will never even attempt to connect directly.

Also, I know for IoT nodes, there was some additional discovery logic that was performed when connection through a tunnel like finding the hostport the reverse tunnel system was running on, any changes needed on that front to support Kubernetes?